### PR TITLE
Added `quiet` to some CLI profiles

### PR DESCRIPTION
**Pull request by caydey**
"quiet" added to these profiles
- bleachbit
- img2txt
- keepassxc-cli
- ncdu
- pdftotext
- weechat

### CODE REVIEW

File: etc/profile-m-z/ncdu.profile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Firejail profile for ncdu
 # Description: Ncurses disk usage viewer
 # This file is overwritten after every install/update
+quiet
```
Comment (rusty-snake) [inline etc/profile-m-z/ncdu.profile:4]: Add `quiet`or `ignore quiet` to ncdu2.profile.

File: etc/profile-m-z/weechat.profile [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Firejail profile for weechat
 # Description: Fast, light and extensible chat client
 # This file is overwritten after every install/update
+quiet
```
Comment (rusty-snake) [inline etc/profile-m-z/weechat.profile:4]: Add `quiet`or `ignore quiet` to weechat-curses.profile.

**Review by rusty-snake — COMMENTED** [review]
LGTM

**Review by kmk3 — CHANGES_REQUESTED** [review]
> [Merge branch 'netblue30:master' into master](https://github.com/netblue30/firejail/pull/4438/commits/2f6d71b7a570e9a142266eed1531c2dccc0dada1)

Please refrain from pulling changes from master after a PR is opened unless
they are substantial to the PR.  For example, when conflicts need to be
resolved or if a change impacts your PR.  A merge-based pull makes the history
more complicated and a rebase-based pull can make reviewing harder.

And if you are going to pull from master, in most cases it's better to do so
with rebase rather than merges (unless you have a very good reason to use
merges).  Example:

```sh
# ensure that there are no outstanding commits
git checkout mybranch
git push
# update master
git checkout master
git pull
# rebase to master
git checkout mybranch
git rebase master
```

With shortcuts:


```sh
# ensure that there are no outstanding commits
git checkout mybranch
git push
# update/rebase to master
git pull --rebase origin master
```

Since you're developing this direclty on your master branch:

```sh
# ensure that there are no outstanding commits
git push
# update/rebase to master
git pull --rebase netblue30 master
```

I'd suggest doing the above on this branch, as rebasing by default flattens the
merges/makes the history linear.

Also, I just noticed that this branch contains a duplicate commit from the
upstream master (from #4434):

Original:

* commit 242d5c764 ("Added ~/Private blacklist (`#4434`)")

Duplicate:

* commit 4bc847151 ("Added ~/Private blacklist")

This is one reason why each PR should be developed in its own separate topic
branch.  I'd advise to never commit directly on master unless you know enough
git to identify and prevent problems such as the above one.

**Comment by rusty-snake** [comment]
@caydey

**Comment by netblue30** [comment]
Let's grab it as is, thanks @caydey for the patch! I'll clean up the rest of them.

**Comment by netblue30** [comment]
added quiet to ncdu2 and weechat-curses in the next commit.

**Review by Lealed02 — APPROVED** [review]

**Pull request merged**